### PR TITLE
Cd/add metrics logger

### PR DESCRIPTION
**Pull request by chribsen**

### CODE REVIEW

**Comment by chribsen** [comment]
@mathiastj do we have any node 6 projects that we should be compatible with?

**Comment by mathiastj** [comment]
Should probably update the README to show how to use the new format.
Also create a test that checks that the it logs to the console what we expect it to. This is also needed to bring coverage back to 100% (use nyc locally to figure out which lines are missing tests)

**Comment by chribsen** [comment]
Just talked to @mathiastj : It probably makes sense to make `gauge()` log the data point immediately as it represents a given point in time

File: src/metric.js [hunk 1]
```diff
@@ -63,18 +70,25 @@ class MetricRegistry {
 
     return name
   }
-  async gauge(name, value, labels) {
+  async gauge(name, value, labels, reducerFn = null) {
     const key = this.createKey(name, labels)
+    const metric = this.metrics[key]
```
Comment (mathiastj) [inline src/metric.js]: Make sure to be consistent, i.e. in the cumulative function too

**Pull request merged**